dist/stat_data.py

```diff
@@ -1060,9 +1060,11 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Checkpoint statistics
     ##########################################
-    CheckpointStat('checkpoint_cleanup_pages_evict', 'pages added for eviction during checkpoint cleanup'),
-    CheckpointStat('checkpoint_cleanup_pages_removed', 'pages removed during checkpoint cleanup'),
-    CheckpointStat('checkpoint_cleanup_pages_visited', 'pages visited during checkpoint cleanup'),
+    CheckpointStat('cc_obsolete_timewindow_pages_read', 'obsolete time window pages read into the cache during checkpoint cleanup tree walk'),
+    CheckpointStat('cc_pages_evict', 'pages added for eviction during checkpoint cleanup'),
+    CheckpointStat('cc_pages_obsolete_timewindow', 'pages dirtied to remove obsolete time window during checkpoint cleanup'),
+    CheckpointStat('cc_pages_removed', 'pages removed during checkpoint cleanup'),
+    CheckpointStat('cc_pages_visited', 'pages visited during checkpoint cleanup'),
     CheckpointStat('checkpoint_cleanup_pages_walk_skipped', 'pages skipped during checkpoint cleanup tree walk'),
```

Comment:
Should this use the cc prefix as well? Should we have a seperate category for checkpoint cleanup stats now it has its own seperate thread?

Reply:
Yes, missed it. Fixed.